test/index.js

```diff
@@ -27,6 +27,58 @@ test('on() - symbol eventName', async t => {
 	t.deepEqual(calls, [1, 2]);
 });
 
+test.cb('on() - listenerAdded', t => {
```

Comment:
Generally, don't use `test.cb`. There's usually a better solution. Here, you could use https://github.com/sindresorhus/p-event

Reply:
Fixed!